PyRDF/backend/Dask.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import print_function
+
+import logging
+from pprint import pformat
+
+from PyRDF.backend.Dist import Dist
+
+import dask
+from dask.distributed import Client
+
+logger = logging.getLogger(__name__)
+
+
+class Dask(Dist):
+    """Dask backend for PyRDF."""
+
+    MIN_NPARTITIONS = 2
+
+    def __init__(self, config={}):
```

Comment:
needs docs for what parameters can be passed as configuration

Reply:
Sure! I'll make more tests and finalize the PR with docs in the end :+1: